tests/testthat/test_civis_ml_utils.R

```diff
@@ -106,3 +106,24 @@ test_that("get_train_template_id reverts to last id if others not available", {
   expect_equal(id, ans)
 })
 
+test_that("coef.civis_ml returns correct coefficients", {
+  lr <- civis_ml(mtcars,
+                        dependent_variable = "vs",
+                        model_type = "sparse_logistic")
```

Comment:
I think this will fail in the test environment because there isn't a `CIVIS_API_KEY`.  Instead of training a new model here, you should be able to use one of the model from `models_list`, which is defined on Line 4 above.

Reply:
You're totally right! Thanks Keith. That is what I ended up doing in the latest commit.